docs/urls.md

```diff
@@ -18,11 +18,15 @@ The views would then acessible as:
 - `/api/auth/login` -> `LoginView`
 - `/api/auth/logout` -> `LogoutView`
 - `/api/auth/logoutall` -> `LogoutAllView`
+- `/api/auth/refresh` -> `TokenRefreshView`
+- `/api/auth/refresh/all` -> `TokenRefreshAllView`
 
 they can also be looked up by name:
 
 ```python
 reverse('knox_login')
 reverse('knox_logout')
 reverse('knox_logoutall')
+reverse('knox_refresh')
+reverse('knox_refresh_all')
```

Comment:
Goes with my comment above: `knox_logout_all` => `knox_logout_all`

Reply:
I edited your comment, think you typed before and after as the same :) It's weird that github allows one to edit other people's comments I feel.